src/lib/singelton/loggerService.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger, ETransportType, ELoglevel } from "../logger/index";
+
+export class LoggerSing extends Logger
+{
+    private static _instance: LoggerSing;
+
+    public constructor(loglevel)
```

Comment:
make the constructor private otherwise it is no singleton

Reply:
removed Singolton, fixed with https://github.com/senseexcel/ser-ext-ondemand/commit/39ff29d840b6de7135be57615e679bf72f18be35